src/components/molecules/NavBar/NavBar.scss

```diff
@@ -1,6 +1,7 @@
 @import "scss/constants.scss";
 
-$border-radius: 28px;
+$menu-item-size: 32px;
+$logo-size: 43px;
```

Comment:
[OOS]

Maybe it's OK to deviate a pixel or two if that would make them more inline with `$spacing--*` variables which are roughly powers of 2 or at least even numbers: `4px`, `8px`, `16px`...

I would suggest minor changes in the direction of `40px` or `44px` when encountering this kind of values in the future.

Reply:
fixed 433dfe8